iFixit/react-commerce
# Add product page faq section

closes #1541 
~~Blocked by #1542~~

## QA

1. Open [featured product page preview](https://react-commerce-git-add-product-page-faq-section-ifixit.vercel.app/products/repair-business-toolkit)
2. Verify that the FAQs section is rendered according to #1541 
3. Verify that the section is responsive
4. Verify that the section content can be changed from [Strapi](https://add-product-page-faq-section.govinor.com/admin/content-manager/collectionType/api::product.product/1) (you might need to wait a little bit for the Redis cache to expire or drop the cache entry from `nextjs-dev` db in Upstash)
